### ISSUE TITLE
wavetable piano 3, power-of-2 error, freeze

### ISSUE THREAD

**Issue by cxbignekoc** [surge-synthesizer/surge#1246]
Found a wavetable issue with error and freeze plugin by switching wavetables:
**Message:**

> Sorry, Surge only supports power-of-2 samplecount wavetables up to 4096 with at least 3 loops at this time. You provided a wavetable with 1 loop of 1024 samples.

Place:
Wavetable Folder ->sampled
- piano 3

__
surge V.1.6.2.1 - 64bit - VST3
Bitwig 3.0.2
Win 7 64 bit

**Comment by baconpaul** [comment]
Thank you - will look

**Comment by baconpaul** [comment]
So I see the error with the wavetable but I don't see the freeze
But in any case Piano 3 isn't in a format surge can handle (a one loop 22khz mono sample)
So my solution to this issue is just to remove the .wav file from the factory distribution which I'll do.

I'm a bit concerned about the freeze though. If you click "OK" on the error box does the system not return to normal?

**Comment by cxbignekoc** [comment]
dont know if that mail comes to you:

no, the plugin need complete to close, bitwig shows error box, but biwig sandboxes that, so that i need to restart the plug.

but switching again the wavetables makes the same issue...i will cancel that wav...maybe a package/ download/ something failure...

regards

**Comment by baconpaul** [comment]
Yeah the .wav file was non-loadable and shouldn’t be in the distro. I removed it from the repo last night so the current nightly and next release won’t include it. .wav files need specific format to load. 

The hang is a bit concerning though. I’ll leave this issue open until the next time I start up a windows VM and see if I can see why that popup makes bitwig windows hang. On macOS I dismissed the dialog and the synth continued as expected. I’m going to rename the issue to indicate that the hang is the current remaining issue also.

Thanks for the report.
